src/main/java/com/parasoft/findings/sonar/rules/LanguageRules.java

```diff
@@ -51,4 +52,12 @@ public int hashCode()
     {
         return language.hashCode();
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if(!(obj instanceof LanguageRules)){
+            return false;
+        }
+        return Objects.equals(((LanguageRules) obj).language, language);
```

Comment:
Do we have valid reason for not considering other variables like "repositoryId" in `equals`?

Reply:
> Do we have valid reason for not considering other variables like "repositoryId" in `equals`?

The existing `hashCode` method above use `language` property only.

Tried to install the plugin before and after these changes on SonarQube 9.9, the number of rules for both Java and C# are the same(3340 and 2362).